src/Tizen.NUI/src/public/AccessibilityManagerEvent.cs

```diff
@@ -795,6 +938,14 @@ internal partial class AccessibilityManager
             }
         }
 
+        /// <summary>
+        /// This is emitted when accessibility action is received to focus and read from the first item on the top continuously
+        /// (by three fingers single tap).
+        /// </summary>
+        /// <returns> The signal to connect to </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Could you check this Event code pattern ? this is old style. you can refer the View event code... => tizenfx/src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

=> no lock(), added some condition to check native signal status.

Reply:
Removed 'lock()' in all signals and changed the event code pattern.
I refer to ViewEvent.cs and FocusManager.cs  code.